### ISSUE TITLE
java.lang.IllegalAccessException: Tried to access visual service WindowManager from a non-visual Context. 

### ISSUE THREAD

**Issue by brewin** [microsoft/appcenter-sdk-android#1427]
### **Description**

```
D/StrictMode: StrictMode policy violation: android.os.strictmode.IncorrectContextUseViolation: Visual services, such as WindowManager, WallpaperService or LayoutInflater should be accessed from Activity or other visual Context. Use an Activity or a Context created with Context#createWindowContext(int, Bundle), which are adjusted to the configuration and visual bounds of an area on screen.
        at android.os.StrictMode.onIncorrectContextUsed(StrictMode.java:2172)
        at android.app.ContextImpl.getSystemService(ContextImpl.java:1914)
        at android.content.ContextWrapper.getSystemService(ContextWrapper.java:803)
        at com.microsoft.appcenter.utils.DeviceInfoHelper.getScreenSize(DeviceInfoHelper.java:152)
        at com.microsoft.appcenter.utils.DeviceInfoHelper.getDeviceInfo(DeviceInfoHelper.java:99)
        at com.microsoft.appcenter.channel.DefaultChannel.enqueue(DefaultChannel.java:634)
        at com.microsoft.appcenter.AppCenter.sendStartServiceLog(AppCenter.java:998)
        at com.microsoft.appcenter.AppCenter.finishStartServices(AppCenter.java:984)
        at com.microsoft.appcenter.AppCenter.access$700(AppCenter.java:62)
        at com.microsoft.appcenter.AppCenter$8.run(AppCenter.java:887)
        at android.os.Handler.handleCallback(Handler.java:938)
        at android.os.Handler.dispatchMessage(Handler.java:99)
        at android.os.Looper.loop(Looper.java:223)
        at android.os.HandlerThread.run(HandlerThread.java:67)
     Caused by: java.lang.IllegalAccessException: Tried to access visual service WindowManager from a non-visual Context. 
        at android.app.ContextImpl.getSystemService(ContextImpl.java:1913)
        at android.content.ContextWrapper.getSystemService(ContextWrapper.java:803) 
        at com.microsoft.appcenter.utils.DeviceInfoHelper.getScreenSize(DeviceInfoHelper.java:152) 
        at com.microsoft.appcenter.utils.DeviceInfoHelper.getDeviceInfo(DeviceInfoHelper.java:99) 
        at com.microsoft.appcenter.channel.DefaultChannel.enqueue(DefaultChannel.java:634) 
        at com.microsoft.appcenter.AppCenter.sendStartServiceLog(AppCenter.java:998) 
        at com.microsoft.appcenter.AppCenter.finishStartServices(AppCenter.java:984) 
        at com.microsoft.appcenter.AppCenter.access$700(AppCenter.java:62) 
        at com.microsoft.appcenter.AppCenter$8.run(AppCenter.java:887) 
        at android.os.Handler.handleCallback(Handler.java:938) 
        at android.os.Handler.dispatchMessage(Handler.java:99) 
        at android.os.Looper.loop(Looper.java:223) 
        at android.os.HandlerThread.run(HandlerThread.java:67) 
```
### **Repro Steps**
Call AppCenter.start() in Application with StrictMode enabled

### **Details**

1. Which SDK version are you using?
    - 3.2.2
2. Which OS version did you experience the issue on?
    - Android 11 (API 30)
3. What device version did you see this error on?  Were you using an emulator or a physical device?
    - Google Pixel 3 emulator

**Comment by russelarms** [comment]
Hi @brewin, thanks for pointing that out!

As far as I can see the _IncorrectContextUseViolation_ warning has been introduced in Android 11.
It doesn't prevent the app from running successfully on Android 11 at the moment, but we'll investigate it.

**Comment by AnastasiaKubova** [comment]
The fix has been released in [3.3.0](https://github.com/microsoft/appcenter-sdk-android/releases/tag/3.3.0).
